platform/ios/PolyPodApp/FeatureContainer/initIframe.js

```diff
@@ -1,13 +1,11 @@
-const {port1, port2} = new MessageChannel();
+const { port1 } = new MessageChannel();
```

Comment:
IIRC, the second port was needed for receiving responses from the platform in the feature. So if that's true, if my changes for #1144 are pulled in, they should trigger a test failure, since the _test_ feature does cover that already.

Reply:
But is it needed *here*? The other sentence I've deleted "transfers" ownership to that port. Do we need to do that? Is that checked by the test feature?

Reply:
I vaguely recall so, but it's been too long that I dove into that code... You also need to look into the feature side of this, i.e. bootstrap.js - there this message is probably expected. I vaguely recall that we use it to wait for the pod to initialise, so that wouldn't work...

I see only two ways forward though:

1. We change it and do _thorough_ manual testing (pretty much all states of all features)
2. We actually _understand_ what happens here, make sure broken logic would be covered by automated tests (_test_ feature most likely), and only change it then

Reply:
But I prefer 3 - to just leave it alone for now :P

Reply:
We need to tackle this sooner or later, though...